pkg/controllers/securitypolicy_controller.go

```diff
@@ -121,7 +121,7 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	} else {
 		if controllerutil.ContainsFinalizer(obj, util.FinalizerName) {
 			metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerDeleteTotal, METRIC_RES_TYPE)
-			if err := r.Service.DeleteSecurityPolicy(obj.UID); err != nil {
+			if err := r.Service.DeleteSecurityPolicy(types.UID(obj.UID)); err != nil {
```

Comment:
Just curious that why it's `(types.UID(obj.UID))`, not `(obj.UID)`?

Reply:
Yes, it's redundant, changed.